libs/ssl-config/src/main/java/org/opensearch/common/ssl/SslConfigurationLoader.java

```diff
@@ -360,14 +360,11 @@ private <V> List<V> resolveListSetting(String key, Function<String, V> parser, L
 
     private static List<String> loadDefaultCiphers() {
         final boolean has256BitAES = has256BitAES();
-        final boolean tlsV13Supported = DEFAULT_PROTOCOLS.contains("TLSv1.3");
         List<String> ciphers = new ArrayList<>();
-        if (tlsV13Supported) { // TLSv1.3 cipher has PFS, AEAD, hardware support
```

Comment:
Why does this PR make this check unnecessary?

Reply:
Ignore this comment, I can see how its always added in SSLConfiguration now so it will never be false.